src/content/admin/resource-manager.mdx

```diff
@@ -0,0 +1,127 @@
+1. Toggle the switch to "Enable". This will activate the Resource Manager without needing to manually update the Helm chart
+
+[Add Screenshot here]
+
+Once enabled, the Resource Manager will immediately begin monitoring and adjusting resource requests based on your specified schedule.
+
+### Disabling the Resource Manager
+If you need to disable the Resource Manager temporarily, you can do so either by:
+
+- Using the toggle switch in the Admin UI to dynamically disable the feature without redeploying the chart
+- Removing or setting `enabled: false` in the Helm values file and redeploying your Helm chart
+
+### Customizing Resource Manager Settings
+You can customize the Resource Manager by adjusting the following settings in your Helm values file:
+
+- `schedule`: Frequency (in seconds) at which the Resource Manager runs
+- `deletePeriod`: Number of days to retain resource data before deletion
+- `priorityClassName`: Assigns a priority class to the Resource Manager’s pods
+- `resources`: Defines resource requests and limits for the Resource Manager itself
+
+## Viewing Resource Metrics and Recommendations
+
+### Resource Metrics View
+
+The Resource Manager introduces a new "Metrics" tab in the Okteto Dashboard, designed to give you clear insights into the resource usage and performance of your Kubernetes workloads.
+This tab is available for key resources, including Deployments, StatefulSets, Jobs, CronJobs, and Pods.
+
+#### Accessing the Metrics Tab
+
+#### What You'll See in the Metrics Tab
+
+- **Average CPU and Memory Usage**: Displays metrics for your average CPU and memory consumption. This helps you understand how resources are being utilized over time
+
+[Screenshot]
+
+- **Resource Manager Recommendations**: If the Resource Manager identifies opportunities for optimizing resource requests, it will display a recommendation.
+These recommendations might include increasing or decreasing CPU and memory requests based on the actual usage patterns
+
+[Screenshot]
+
+### Recommendations Banner
+If the Resource Manager detects potential improvements, a banner will appear in the Okteto Dashboard. This banner will:
+
+- Highlight performance issues or inefficiencies
+- Provide links to the Metrics tab where you can view detailed recommendations
+- Encourage enabling the Resource Manager if it is currently disabled
+
+The banner encourages you to review these recommendations and, if the Resource Manager is not enabled, to activate it for automated management.
+You can interact with the banner to view a detailed list of resources with recommendations.
+
+[Screenshot]
+
+## Rolling Back Changes
+If you find that the Resource Manager's recommendations are not suitable for your workload, you can override its behavior:
+
+Add the annotation `dev.okteto.com/resource-manager: false` to your pod's metadata to prevent automatic adjustments.
```

Comment:
I would put this info in another place. Maybe in this section:
https://github.com/okteto/docs/pull/826/files#diff-fb4aca11868f791cc679c3231c9a7ef5ee5f81ead783ba4d60a6d51141e14e4dR61
We could even have a FAQ explaining how to disable the RM for a particular service

Reply:
I think a dedicated section would be helpful for this - I imagine questions on how to do this will come up initially. Could you help me with the content for that @pchico83 ?

Reply:
do you mean to explain how to add the annotation?